shared/wallets/send-form/participants/container.js

```diff
@@ -23,6 +23,7 @@ const mapStateToProps = (state: TypedState) => {
     .toArray()
   let fromAccount
   let toAccount
+  let showSpinner = state.wallets.waitingForSetBuildingTo
```

Comment:
`Constants.linkExistingWaitingKey` and `Constants.createNewAccountWaitingKey` exist, could this instead be an `anyWaiting` on those keys and the one for `loadAccounts`?

Reply:
I'm a little worried about having `waitingForSetBuildingTo` as a separate flag. If one of the RPCs in the createAccount -> setBuildingTo fails it could get into the wrong state

Reply:
Will do; I guess I just didn't read enough about existing types.